<launch>

	<arg name="noise_scaling" default="0.00"/>

	<param name="noise_scaling" type="double" value="$(arg noise_scaling)"/>
    <node pkg="memory_visualizer" type="memory_visualizer_node" name="state_estimate_corruptor" output="screen">
    </node>
   
</launch>
